### ISSUE TITLE
Substitute autoconf pkgdatadir variable in arp-scan.1 manpage

### ISSUE THREAD

**Issue by royhills** [royhills/arp-scan#77]
The `arp-scan.1` manpage contains fixed pathnames for files in datadir, e.g. `/usr/local/share/arp-scan/ieee-oui.txt`.

These hardcoded path names are not necessarily correct on the end-user's system because they depend on the configure prefix variable which can be set to different values depending on the OS distro. e.g. `/usr/local/share/arp-scan` on some systems and `/usr/share/arp-scan` on others.

We should substitute the appropriate path name into the manpage after configure has been run.

**Comment by royhills** [comment]
hopefully addressed by commit ad835ff87a99f4c17bfdf60bd50095cb2e19ed6e

**Comment by royhills** [comment]
This appears to be working.

Looking at the loction of mac-vendor.txt in files section of the arp-scan.1 manpage:

With `$ ./configure` (no options specified):
```
       /usr/local/share/arp-scan/mac-vendor.txt
              List of other Ethernet MAC to vendor mappings.
```

With `$ ./configure --prefix=/usr --sysconfdir=/etc --localstatedir=/var` (like a debian package build)

```
       /usr/share/arp-scan/mac-vendor.txt
              List of other Ethernet MAC to vendor mappings.
```

Marking this as closed now.
